packages/apps/human-app/frontend/src/pages/worker/send-reset-link/send-reset-link-success.page.tsx

```diff
@@ -27,7 +27,7 @@ export function SendResetLinkWorkerSuccessPage() {
             i18nKey="worker.sendResetLinkSuccess.paragraph1"
             values={{ email }}
           >
-            Strong <Typography variant="buttonMedium" />
+            Strong <Typography fontWeight={600} />
```

Comment:
If the given font is not defined in Figma typography just add it as the next `body<index>`  font in `theme.ts` - `body7` probably